diff --git a/src/DensityGrid.cpp b/src/DensityGrid.cpp
index 466f5ded3..624e196d7 100644
--- a/src/DensityGrid.cpp
+++ b/src/DensityGrid.cpp
@@ -69,7 +69,7 @@ void DensityGrid::Init() {
         Density = new float[GRID_SIZE][GRID_SIZE];
         fall_off = new float[RADIUS * 2 + 1][RADIUS * 2 + 1];
         Bins = new deque<Node>[GRID_SIZE * GRID_SIZE];
-    } catch (bad_alloc errora) {
+    } catch (bad_alloc&) {
         // cout << "Error: Out of memory! Program stopped." << endl;
 #ifdef MUSE_MPI
         MPI_Abort ( MPI_COMM_WORLD, 1 );
diff --git a/src/DensityGrid_3d.cpp b/src/DensityGrid_3d.cpp
index cc321f9f6..93d3d8cc9 100644
--- a/src/DensityGrid_3d.cpp
+++ b/src/DensityGrid_3d.cpp
@@ -69,7 +69,7 @@ void DensityGrid::Init() {
         Density = new float[GRID_SIZE][GRID_SIZE][GRID_SIZE];
         fall_off = new float[RADIUS * 2 + 1][RADIUS * 2 + 1][RADIUS * 2 + 1];
         Bins = new deque<Node>[GRID_SIZE * GRID_SIZE * GRID_SIZE];
-    } catch (bad_alloc errora) {
+    } catch (bad_alloc&) {
         // cout << "Error: Out of memory! Program stopped." << endl;
 #ifdef MUSE_MPI
         MPI_Abort ( MPI_COMM_WORLD, 1 );
