diff --git a/cigraph/interfaces/functions.def b/cigraph/interfaces/functions.def
index 324a7f5b0..7431c78b3 100644
--- a/cigraph/interfaces/functions.def
+++ b/cigraph/interfaces/functions.def
@@ -248,7 +248,7 @@ igraph_asymmetric_preference_game:
         IGNORE: RR, RC, RNamespace
 
 igraph_rewire_edges:
-        PARAMS: INOUT GRAPH graph, REAL prob, BOOLEAN loops=False
+        PARAMS: INOUT GRAPH graph, REAL prob, BOOLEAN loops=False, BOOLEAN multiple=False
         IGNORE: RR, RC, RNamespace
 
 igraph_watts_strogatz_game:
@@ -1358,8 +1358,7 @@ igraph_community_leading_eigenvector:
 igraph_community_fluid_communities:
         PARAMS: GRAPH graph, INTEGER no_of_communities, \
 		            OUT VECTOR membership, OUT REALPTR modularity
-        NAME-R: cluster_fluid_com
-        IGNORE: RR
+        NAME-R: cluster_fluid_communities
 
 igraph_community_label_propagation:
         PARAMS: GRAPH graph, OUT VECTOR membership, EDGEWEIGHTS weights=NULL, \
