--- a/src/foreign-dl-lexer.c	2021-09-16 21:18:05.000000000 +0200
+++ b/src/foreign-dl-lexer	2021-09-16 21:17:12.000000000 +0200
@@ -1864,7 +1864,6 @@
 static void yy_fatal_error (yyconst char* msg , yyscan_t yyscanner)
 {
     	(void) fprintf( stderr, "%s\n", msg );
-	exit( YY_EXIT_FAILURE );
 }
 
 /* Redefine yyless() so it works in section 3 code. */
--- a/src/foreign-gml-lexer.c	2021-09-16 21:18:05.000000000 +0200
+++ b/src/foreign-gml-lexer	2021-09-16 21:17:12.000000000 +0200
@@ -1684,7 +1684,6 @@
 static void yy_fatal_error (yyconst char* msg , yyscan_t yyscanner)
 {
     	(void) fprintf( stderr, "%s\n", msg );
-	exit( YY_EXIT_FAILURE );
 }
 
 /* Redefine yyless() so it works in section 3 code. */
--- a/src/foreign-lgl-lexer.c	2021-09-16 21:18:05.000000000 +0200
+++ b/src/foreign-lgl-lexer	2021-09-16 21:17:12.000000000 +0200
@@ -1643,7 +1643,6 @@
 static void yy_fatal_error (yyconst char* msg , yyscan_t yyscanner)
 {
     	(void) fprintf( stderr, "%s\n", msg );
-	exit( YY_EXIT_FAILURE );
 }
 
 /* Redefine yyless() so it works in section 3 code. */
--- a/src/foreign-ncol-lexer.c	2021-09-16 21:18:05.000000000 +0200
+++ b/src/foreign-ncol-lexer	2021-09-16 21:17:12.000000000 +0200
@@ -1638,7 +1638,6 @@
 static void yy_fatal_error (yyconst char* msg , yyscan_t yyscanner)
 {
     	(void) fprintf( stderr, "%s\n", msg );
-	exit( YY_EXIT_FAILURE );
 }
 
 /* Redefine yyless() so it works in section 3 code. */
--- a/src/foreign-pajek-lexer.c	2021-09-16 21:18:05.000000000 +0200
+++ b/src/foreign-pajek-lexer	2021-09-16 21:17:12.000000000 +0200
@@ -2099,7 +2099,6 @@
 static void yy_fatal_error (yyconst char* msg , yyscan_t yyscanner)
 {
     	(void) fprintf( stderr, "%s\n", msg );
-	exit( YY_EXIT_FAILURE );
 }
 
 /* Redefine yyless() so it works in section 3 code. */
