diff --git a/src/triangles.c b/src/triangles.c
index 2a8037e1f..ea5860bc7 100644
--- a/src/triangles.c
+++ b/src/triangles.c
@@ -31,6 +31,10 @@
 #include "igraph_motifs.h"
 #include "igraph_structural.h"
 
+#ifdef USING_R
+  #include <R.h>
+#endif
+
 /**
  * \function igraph_transitivity_avglocal_undirected
  * \brief Average local transitivity (clustering coefficient).
@@ -484,12 +488,20 @@ int igraph_transitivity_local_undirected(const igraph_t *graph,
     igraph_bool_t simple;
 
     if (igraph_is_directed(graph)) {
+#ifndef USING_R
         IGRAPH_ERROR("Transitivity works on undirected graphs only", IGRAPH_EINVAL);
+#else
+        REprintf("Transitivity works on undirected graphs only. The result might be incorrect if the graph includes mutual edge pairs or multiple edges between the same pair of vertices. igraph 1.3.0 and later will treat this as an error.\n");
+#endif
     }
 
     igraph_is_simple(graph, &simple);
     if (!simple) {
+#ifndef USING_R
         IGRAPH_ERROR("Transitivity works on simple graphs only", IGRAPH_EINVAL);
+#else
+        REprintf("Transitivity works on simple graphs only. The result might be incorrect. igraph 1.3.0 and later will treat this as an error.\n");
+#endif
     }
 
     if (igraph_vs_is_all(&vids)) {
