--- a/src/bliss/graph.cc	2021-09-27 23:23:54.000000000 +0200
+++ b/src/bliss/graph.cc	2021-10-25 17:39:36.000000000 +0200
@@ -1499,6 +1499,9 @@
 	  child_node.cr_cep_stack_size = cr_cep_stack.size();
 	  child_node.cr_cep_index = cr_cep_index;
 
+	  /* Initialize needs_long_prune to prevent a gcc-ubsan warning */
+	  child_node.needs_long_prune = true;
+
 	  search_stack.push_back(child_node);
 	  continue;
 	}
