--- a/src/core/linalg/blas.c
+++ b/src/core/linalg/blas.c
@@ -63,8 +63,13 @@
     IGRAPH_ASSERT(igraph_vector_size(x) == transpose ? m : n);
     IGRAPH_ASSERT(igraph_vector_size(y) == transpose ? n : m);
 
+#ifdef HAVE_GFORTRAN
+    igraphdgemv_(&trans, &m, &n, &alpha, VECTOR(a->data), &m,
+                 VECTOR(*x), &inc, &beta, VECTOR(*y), &inc, /* trans_len = */ 1);
+#else
     igraphdgemv_(&trans, &m, &n, &alpha, VECTOR(a->data), &m,
                  VECTOR(*x), &inc, &beta, VECTOR(*y), &inc);
+#endif
 }
 
 /**
@@ -99,8 +104,13 @@
     m = (int) igraph_matrix_nrow(a);
     n = (int) igraph_matrix_ncol(a);
 
+#ifdef HAVE_GFORTRAN
+    igraphdgemv_(&trans, &m, &n, &alpha, VECTOR(a->data), &m,
+                 (igraph_real_t*)x, &inc, &beta, y, &inc, /* trans_len = */ 1);
+#else
     igraphdgemv_(&trans, &m, &n, &alpha, VECTOR(a->data), &m,
                  (igraph_real_t*)x, &inc, &beta, y, &inc);
+#endif
 }
 
 igraph_real_t igraph_blas_dnrm2(const igraph_vector_t *v) {
